theme/less/_header.less

```diff
@@ -0,0 +1,69 @@
+.header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: #3A7D11;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.png) no-repeat right;
+}
+
+header a
+{
+  text-decoration: none;
+}
+
+.header-top a
+{
+  max-width: 190px;
+}
+
+header p
+{
+  display: inline;
+  opacity: 0.7;
+  color: #000000;
+  padding: 0 10px;
+}
+
+header img
+{
+ vertical-align: middle;
+ width: 32px;
+ height: auto;
+}
```

Comment:
I'm not sure if you intended to apply those styles to all images in the header, or all `p` tags. You can add a custom Id for that reason `#logo`, maybe?

Reply:
done.

Reply:
It seem to be a unique item, so I was suggesting to use id therefore, I think you used class because my suggestion sounded generic, so what about `#navbar-logo` or `#header-logo`? And I meant to apply to the `<a>`, so you can target both `p` and `img` inside.